prow/cmd/tide/config.md

```diff
@@ -141,8 +154,10 @@ tide:
                 - "optional_test"
 ```
 
-**Explanation**: The component starts periodically querying all PRs in `github.com/kubeflow/community` and
-`github.com/kubeflow/examples` repositories that have `lgtm` and `approved` labels set
+**Explanation**: The component starts periodically querying all PRs in `github.com/kubflow` organization excluding `github.com/kubeflow/community`, `github.com/kubeflow/examples` and `github.com/kubeflow/website`repositories that have `lgtm` and `approved` labels set and do not have `do-not-merge` and `do-not-merge/blocked-paths` labels set.
```

Comment:
Not quite correct, it also keeps quering `kubeflow/community` and `kubeflow/examples` that have lgtm+approve but do not have `do-not-merge`, `do-not-merge/hold`, `do-not-merge/work-in-progress`, `needs-ok-to-test` and `needs-rebase`.

Rather than trying to document all possible attributes here, I would prefer if we just kept the simple example and additionally linked to https://github.com/kubernetes/test-infra/blob/27c9a7f2784088c2db5ff133e8a7a1e2eab9ab3f/prow/config/prow-config-documented.yaml#L1208
for a full list of properties. That document is generated from the code and we check that it stays in sync, so that is the better reference IMHO.

Reply:
Thanks a lot for your advice! I'll make some changes later.